daemon/endpoint.go

```diff
@@ -552,7 +570,7 @@ func (d *Daemon) UpdateSecLabels(id string, add, del labels.Labels) middleware.R
 	d.containersMU.RUnlock()
 
 	if !containerFound {
-		return NewPutEndpointIDLabelsNotFound()
+		log.Infof("NewPutEndpointIDLabelsNotFound %v %v container not found", id, ep)
```

Comment:
`log.Infof` with endpoint's `%v` result on an ultra verbose output. Use the relevant endpoint's data or simply the endpoint ID.

If the container was not found shouldn't we `return NewPutEndpointIDLabelsNotFound` like before?